public/coffee/ide/chat/services/chatMessages.coffee

```diff
@@ -47,7 +47,8 @@ define [
 			chat.state.loading = true
 			return $http
 				.get(url)
-				.success (messages = [])->
+				.then (response) ->
+					messages = response.data ? []
```

Comment:
Very minor stylistic comment (no need to change anything), we've tended to use `x || y` in cases of conditional assignment, but I do like the `?` operator better. Maybe we should use this more in future.